src/debugger/breakpoints.jl

```diff
@@ -120,12 +120,14 @@ handle("getBreakpoints") do
   end
 end
 
+isCompileMode() = _compiledMode[] === JuliaInterpreter.Compiled()
+
 handle("toggleCompiled") do
   with_error_message() do
-    _compiledMode[] = (_compiledMode[] === JuliaInterpreter.finish_and_return! ?
-      (_compiledMode[] = JuliaInterpreter.Compiled()) :
-      (_compiledMode[] = JuliaInterpreter.finish_and_return!))
-    return _compiledMode[] === JuliaInterpreter.Compiled()
+    _compiledMode[] = (isCompileMode() ?
+      (_compiledMode[] = JuliaInterpreter.finish_and_return!) :
+      (_compiledMode[] = JuliaInterpreter.Compiled()))
+    return isCompileMode()
```

Comment:
Still now sure why this part wasn't like
```julia
_compiledMode[] = isCompileMode() ? JuliaInterpreter.finish_and_return! : JuliaInterpreter.Compiled()
```

Reply:
Probably because I refactored this at some point and never thought about what I wrote there. Definitely makes more sense like this though ;)

Reply:
fixed ;)